torch/distributed/checkpoint/optimizer.py

```diff
@@ -169,19 +168,9 @@ def create_local_plan(self) -> LoadPlan:
 
             assert len(obj.local_shards()) == 1
             original_shard = obj.local_shards()[0]
-            local_chunks = [
-                ChunkStorageMetadata(
-                    offsets=torch.Size(
-                        _element_wise_add(
-                            original_shard.metadata.shard_offsets, offset
-                        )
-                    ),
-                    sizes=torch.Size(original_shard.metadata.shard_sizes),
-                )
-            ]
-
-            reqs = create_read_items_for_chunk_list(
-                fqn, cast(TensorStorageMetadata, md), local_chunks
+            shard_md = copy.deepcopy(original_shard.metadata)
```

Comment:
@kumpera Do we want to make this change in the main branch first? 
https://github.com/pytorch/pytorch/blob/main/torch/distributed/checkpoint/optimizer.py#L170-L183